### DISCUSSION TITLE
If OSU!(lazer) put any temp files on C:/,and how to delete them

### DISCUSSION THREAD

**Discussion by guanjiao23456** [ppy/osu#28756] [Q&A]
sorry i am not good at english
i had move osu from C:/ to other place,but when i run osu,C:/ alway be filled
i want to know if it be caused by osu and how to solve

**Comment by CloneWith** [comment]
Almost yes.

Did you use the in-game option to move osu!lazer? Files of osu!lazer are stored in the following locations:

- C:\\Users\\\<Username\>\\AppData
  - \\Local\\osulazer: Basic program files, where all executables exist. osu!lazer won't move them.
  - \\Roaming\\osu: Your data files (including beatmaps, skins, etc.)
- Other places (if you used the in-game option to change the directory)

> i had move osu from C:/ to other place,but when i run osu,C:/ alway be filled

Chances are that osu!lazer is downloading updates in the basic directory. It's better to leave some extra space for osu!, and not recommended to move them by yourself.

**Comment by Stoppedpuma** [comment]
> i run osu,C:/ alway be filled

Keeping the folder on the install drive is intentional as it creates a file called `storage.ini` which points to the user defined directory when the change folder location option is used. All existing beatmaps, skins, etc will be moved when using that option as well as new files.
